src/lib/geocoding-api/country-codes.ts

```diff
@@ -163,6 +163,7 @@ const countryNameToTwoLetterIsoCountryCodeMap: Record<string, TwoLetterIsoCountr
   "Northern Mariana Islands (Commonwealth of the)": "MP",
   Norway: "NO",
   "occupied Palestinian territory, including east Jerusalem": "PS",
+  "occupied Palestinian territory - including east Jerusalem": "PS",
```

Comment:
Some names are a tiny bit different, I'll need to update the front end too unfortunately just because we haven't made it agnostic of these names yet.